q2_fondue/tests/test_scraper.py

```diff
@@ -32,6 +38,17 @@ def _open_json_file(self, filename):
         file = open(path2col)
         return json.load(file)
 
+    def _open_txt_file(self, filename):
+        path2file = self.get_data_path(filename)
+        txt_file = open(path2file, "r")
+        return json.loads(txt_file.read())
```

Comment:
Seeing this made me realize that all your Zotero test files are actually `json` files - please change extensions of all these to 'json' and adjust the name of this function to indicate that.